src/BinaryBuilder.jl

```diff
@@ -10,4 +10,24 @@ include("Auditor.jl")
 include("DockerRunner.jl")
 include("Dependency.jl")
 
-end # module
\ No newline at end of file
+export shortname
+
+"""
+    shortname(p::Platform)
+
+Get a convenient symbol representation of the given platform.
+
+# Examples
+```jldoctest
+julia> shortname(Linux(:i686))
+:linux32
+
+julia> shortname(MacOS())
+:osx64
+```
```

Comment:
I like doc tests. I forgot they exist. How do I run the doctests?

Reply:
Well, first you need Documenter. Is that set up here?